ArduPlane/Parameters.cpp

```diff
@@ -1247,6 +1247,11 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_SUBGROUPINFO(efi, "EFI", 22, ParametersG2, AP_EFI),
 #endif
 
+#if OFFBOARD_GUIDED == ENABLED
+    // @Path: ../libraries/AC_PID/AC_PID.cpp
```

Comment:
Missing the @ Group tag here.

Reply:
ok, will fix in next push.    not entirely sure what this does, and the existing code seems to be very inconsistent with regard to whether to have a trailing _ or not, so i'll have a 50% chance of getting it right , if it matters.